python/ray/autoscaler/tags.py

```diff
@@ -13,6 +13,9 @@
 # Tag for user defined node types (e.g., m4xl_spot). This is used for multi
 # node type clusters.
 TAG_RAY_USER_NODE_TYPE = "ray-user-node-type"
+# Tag for autofilled node types for legacy cluster yamls without multi
+# node type defined in the cluster configs.
+TAG_RAY_LEGACY_NODE_TYPE = "ray-legacy-node-type"
```

Comment:
```suggestion
NODE_TYPE_LEGACY_NODE_TYPE = "ray-legacy-node-type"
```

(it's not a tag name, but a tag value)